services/order.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    try:
+        user = get_user_model().objects.get(username=username)
+    except User.DoesNotExist:
+        raise ValueError(f"There is no user with username '{username}'")
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+
```

Comment:
In case all body of a method should be treated as an atomic transaction, it is better to use transaction.atomic as a decorator